config.tpl.yml

```diff
@@ -41,6 +38,14 @@ APPS:
           weekends:
             - 08:00-23:00
 
+  - name: notify-api-sms-callbacks
+    min_instances: {{ MAX_INSTANCE_COUNT_MEDIUM }}
+    max_instances: {{ MAX_INSTANCE_COUNT_API }}
```

Comment:
How did you decide that the min instances should be 10, max instances should be 35 and the threshold should be 250?

Some stats for you from load testing which may be useful. 10 instances looks to be able to handle with good quick response times 500 requests per second. We haven't tested it on higher numbers of requests per second but given the response times were good, its likely that this app won't ever really need to scale above 15 instances (it may not even need that many, the load test didn't tell us the breaking point of 10 instances, they just told us that they can handle 500 requests per second, 10 instances might be able to handle 800 rps for example).

Reply:
In that case do we need to scale this app at all?

Reply:
I think we could go for something simple, in particular given that we know SMS sending will be predominantly during daylight hours. I think we should stick with 10 instances during the day but scale down to say 4 overnight using the `ScheduleScaler`. 

1 instance of this 1GB app costs about £9 per month to run. if we run 10 instances of this app for a year that will cost about £1080 per year.

If we run 4 instances for half the day, and 10 instances for half the day that changes to cost about £756 per year.

300 quid saved. Up to you if you think it's worth the hassle of figuring it out the autoscaling configuration.

I'm also fine if you just decide to close the PR and we can maybe look at scaling again in the future